appeals/web/src/server/appeals/appeal-details/appellant-case/__tests__/__snapshots__/appellant-case.test.js.snap

```diff
@@ -8145,3 +8231,721 @@ exports[`appellant-case POST /appellant-case/invalid should re-render the invali
+            <h1             class="govuk-fieldset__legend--xl govuk-!-margin-top-0">Enter valid date for case</h1>
+                <p class="govuk-body">This is the date all case documentation was received and the appeal was
+                    valid.</p>
+        </div>
+    </div>
+    <form method="post" novalidate="novalidate">
+        <div class="govuk-form-group">
+            <div id="due-date-hint" class="govuk-hint">For example, 27 3 2007</div>
+            <div class="govuk-date-input" id="due-date">
+                <div class="govuk-date-input__item">
+                    <div class="govuk-form-group">
+                        <label class="govuk-label govuk-date-input__label" for="due-date-day">Day</label>
+                        <input class="govuk-input govuk-date-input__input govuk-input--width-2"
+                        id="due-date-day" name="due-date-day" type="text" inputmode="numeric">
+                    </div>
+                </div>
+                <div class="govuk-date-input__item">
+                    <div class="govuk-form-group">
+                        <label class="govuk-label govuk-date-input__label" for="due-date-month">Month</label>
+                        <input class="govuk-input govuk-date-input__input govuk-input--width-2"
+                        id="due-date-month" name="due-date-month" type="text" inputmode="numeric">
+                    </div>
+                </div>
+                <div class="govuk-date-input__item">
+                    <div class="govuk-form-group">
+                        <label class="govuk-label govuk-date-input__label" for="due-date-year">Year</label>
+                        <input class="govuk-input govuk-date-input__input govuk-input--width-4"
+                        id="due-date-year" name="due-date-year" type="text" inputmode="numeric">
+                    </div>
+                </div>
+            </div>
+        </div>
+        <div class="govuk-button-group">
+            <button type="submit" class="govuk-button" data-module="govuk-button">Confirm</button>
+        </div>
+    </form>
+</main>"
+`;
+
+exports[`appellant-case POST /appellant-case/valid/date should re-render the valid date page with the expected error message if provided date is not in the future 1`] = `
+"<main class="govuk-main-wrapper " id="main-content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <div class="govuk-error-summary" data-module="govuk-error-summary">
+                <div role="alert">
+                    <h2 class="govuk-error-summary__title"> There is a problem</h2>
+                    <div class="govuk-error-summary__body">
+                        <ul class="govuk-list govuk-error-summary__list">
+                            <li><a href="#0">The valid date must be on or after the date the case was received.</a>
```

Comment:
Please help me with couple things here:

1. The test description does not match rendered snapshot here?
Description: "error message if provided date is not in the future"
Snapshot error: "The valid date must be on or after the date the case was received"

2. According to BOAT-838 AC the "future" error must appear only when the date is in the future. But according to test description "error message if provided date is not in the future", hence the confusion. Is test checking for future date? Is this a typo in description?